packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxNavItem/OnyxNavItem.vue

```diff
@@ -42,7 +42,7 @@ defineSlots<{
 .onyx-nav-item {
   @include layers.component() {
     .onyx-external-link-icon {
-      margin-left: calc(-1 * var(--onyx-spacing-sm));
+      margin-left: calc(-1 * var(--onyx-menu-item-gap));
```

Comment:
is it certain that OnyxNavItem with external lik icon will never be used without a parent OnyxMenuItem? else, the var would be unresolved right?

Reply:
aah right ^^ thank you :)